packages/p2p/src/components/my-ads/my-ads-row-renderer.jsx

```diff
@@ -245,84 +176,139 @@ const MyAdsRowRenderer = observer(({ row: advert, setAdvert }) => {
                                 </div>
                             )}
                         </div>
-                    </Table.Cell>
-                    <Table.Cell>
-                        {/* TODO: add condition to show/not to show warning icon */}
+                    </Table.Row>
+                }
+            />
+        );
+    }
+
+    return (
+        <div onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave}>
+            <Table.Row
+                className={classNames('p2p-my-ads__table-row', {
+                    'p2p-my-ads__table-row-disabled': !is_advert_active,
+                })}
+            >
+                <Table.Cell>
+                    {is_buy_advert ? (
+                        <Localize i18n_default_text='Buy {{ id }}' values={{ id }} />
+                    ) : (
+                        <Localize i18n_default_text='Sell {{ id }}' values={{ id }} />
+                    )}
```

Comment:
```suggestion
                    {is_buy_advert ? (
                        <Localize i18n_default_text='Buy {{ id }}' values={{ id }} />
                    ) : (
                        <Localize i18n_default_text='Sell {{ id }}' values={{ id }} />
                    )}
```
You can use `value` props for this instead of using 2 Localize component

Reply:
Incorporated the change